product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_an_eha_cluster.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Creating an extreme high-availability cluster
+---
+
+When you create an extreme high-availability cluster, you need to set up the data group. Extreme high-availability clusters can contain one or two data groups. 
+
+1. Sign in to the [BigAnimal](https://portal.biganimal.com) portal.
```

Comment:
I think you could tighten this up by collapsing steps 1 through 5 into this stem sentence:

>After selecting **Extreme High Availability** in the Cluster info tab and specifying your cluster name and password on the Cluster Settings tab:

Reply:
@ebgitelman we haven't been consistent about this, but I think our convention is to not bold tab names. Is that correct? That could be a thing you look out for when you review the BigAnimal docs.

Reply:
I made the above change with bolded tab names and will unbold them if that's not the convention.